main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	registry := os.Getenv("REGISTRY")
+	if registry == "" {
+		log.Fatalf("could not get registry\n")
+	}
+
+	registryOrganisation := os.Getenv("REGISTRY_ORGANISATION")
+	if registryOrganisation == "" {
+		log.Fatalf("could not get registry organisation\n")
+	}
+
+	registryUsername := os.Getenv("REGISTRY_USERNAME")
+	if registryUsername == "" {
+		log.Fatalf("could not get registry username\n")
+	}
+
+	registryPassword := os.Getenv("REGISTRY_PASSWORD")
+	if registryPassword == "" {
+		log.Fatalf("could not get registry password\n")
+	}
+
+	login := exec.Command("docker", "login", "--email=\" \"", "-u", registryUsername, "-p", registryPassword, registry)
+	if err := Run(login); err != nil {
+		log.Fatalf("could not login to registry: %v\n", err)
+	}
+
+	for _, image := range Images {
+		for _, tag := range image.Tags {
+			log.Printf("managing: %v, %v, %v\n", image.Name, tag.Sha, tag.Tag)
```

Comment:
I didn't know this, actually

https://golang.org/pkg/log/#Output
A newline is appended if the last character of s is not already a newline.

That's really useful. I'll strip the newlines.